src/components/modals/WalletListMenuModal.js

```diff
@@ -83,16 +83,33 @@ class WalletListMenuModalComponent extends PureComponent<Props> {
       const { currencyCodes, label, value } = option
       if (currencyCodes != null && !currencyCodes.includes(currencyCode)) continue
 
+      if (option.value === 'split' && this.checkSplitExclude()) continue
+
       const temp = { label, value }
+
       if (option.value === 'split') {
         const splitString = s.strings.string_split_wallet
         const currencyName = currencyCode === 'BTC' ? 'Bitcoin Cash' : 'Bitcoin SV'
         temp.label = sprintf(splitString, currencyName)
       }
+
       this.options.push(temp)
     }
   }
 
+  checkSplitExclude() {
+    const { walletName } = this.props
+
+    if (!walletName) return false
+
+    return (
+      EXCLUDE_SPLIT_WALLETS.findIndex((item: string) => {
```

Comment:
Using "find" makes more sense here than "findIndex", since all you want is a "true/false" value anyway.